phpBB/includes/functions_messenger.php

```diff
@@ -952,7 +952,7 @@ function process()
 							}
 							else
 							{
-								$result = phpbb_mail($to, $subject, $msg, $headers, PHP_EOL, $err_msg);
+								$result = phpbb_mail($to, $subject, $msg, $headers, $encode_eol, $err_msg);
```

Comment:
I am not sure, if this part get's the variable from line 600.

Reply:
It does not.